collectors/statsd.plugin/README.md

```diff
@@ -6,17 +6,19 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/stats
 
 # statsd.plugin
 
-statsd is a system to collect data from any application. Applications are sending metrics to it, usually via non-blocking UDP communication, and statsd servers collect these metrics, perform a few simple calculations on them and push them to backend time-series databases.
+StatsD is a system to collect data from any application. Applications send metrics to it, usually via non-blocking UDP communication, and StatsD servers collect these metrics, perform a few simple calculations on them and push them to backend time-series databases.
+
+If you want to learn more about the StatsD protocol, we have written a [blog post](https://www.netdata.cloud/blog/introduction-to-statsd/) about it!
 
 There is a [plethora of client libraries](https://github.com/etsy/statsd/wiki#client-implementations) for embedding statsd metrics to any application framework. This makes statsd quite popular for custom application metrics.
 
-Netdata is a fully featured statsd server. It can collect statsd formatted metrics, visualize them on its dashboards, stream them to other Netdata servers or archive them to backend time-series databases.
+Netdata is a fully featured statsd server. It can collect statsd formatted metrics, visualize them on its dashboards and store them in it's database for long-term retention.
```

Comment:
I saw that you've made a good effort to standardize the capitalization to **StatsD**, but there's a few missing. Maybe you can do some find-replace? I'm not sure what the negative impact could be.

```suggestion
Netdata is a fully featured StatsD server. It can collect statsd formatted metrics, visualize them on its dashboards, and store them in it's database for long-term retention.
```